package.json

```diff
@@ -36,23 +35,31 @@
     "jsdoc-fresh": "^1.0.2",
     "linkinator": "^1.4.0",
     "mocha": "^6.1.4",
-    "prettier": "^1.17.0"
+    "gts": "^0.9.0",
+    "prettier": "^1.17.0",
+    "typescript": "~3.5.3"
```

Comment:
Latest is on `3.6.4`.  Given the # of repositories here, and the fact that they're generated, I think it may be preferable to use `^`.

Reply:
In https://github.com/googleapis/gapic-generator-typescript/pull/90 I'm setting this to `~3.6.0`. I heard they don't respect semver and can break things, so unsure if we really want to use `^`.